src/boot/sysfail.h

```diff
@@ -6,9 +6,14 @@
 
 typedef enum SysFailType {
         SYSFAIL_OK = 0x0,
```

Comment:
The naming `SYSFAIL_OK` sounds spurious. `FAIL_OK` means succeeds? Or successfully fail??

Reply:
Do you have any suggestions how it might be called in a better way?